src/kontrol/prove.py

```diff
@@ -571,6 +552,19 @@ def _init_cterm(
         init_subst['CALLGAS_CELL'] = intToken(0)
         init_subst['REFUND_CELL'] = intToken(0)
 
+    if symbolic_exploration:
+        init_subst.update(
+            {
+                'CALLSTACK_CELL': KVariable('CALLSTACK'),
+                'CALLDEPTH_CELL': KVariable('CALLDEPTH'),
+                'LOG_CELL': KVariable('LOGS'),
+                'ID_CELL': Foundry.address_TEST_SYMBOLIC(),
+            }
+        )
+        del init_subst['CALLSTACK_CELL']
```

Comment:
I may not fully understand what is being done here, but do we want to delete `CALLSTACK_CELL` and `LOG_CELL` that we've just made symbolic?

Reply:
In the `init_subst` above, `callStack` is initialized with an empty list with `'CALLSTACK_CELL': list_empty(),.`
`list_empty()` is the same as `KApply(.List)`, so the change from `KApply(.List)` to `list_empty()` does not affect the initial configuration in any way.

Deleting the `CALLSTACK_CELL` entry from the `init_subst` will prevent initializing the cell with an empty list and keep it as a symbolic variable.

Reply:
Oh, right, I understood what you mean! Deleting a cell substitution after I replaced it with a symbolic variable was indeed redundant. Thanks!